<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TestingWorld-PC" name="Test" tests="1" failures="1" timestamp="20 Feb 2016 03:54:50 GMT" time="43.753" errors="0">
  <testcase name="testUntitled1" time="32.057" classname="testcases.TC_001">
    <failure type="com.thoughtworks.selenium.SeleniumException" message="Timed out after 30000ms">
      <![CDATA[com.thoughtworks.selenium.SeleniumException: Timed out after 30000ms
	at com.thoughtworks.selenium.HttpCommandProcessor.throwAssertionFailureExceptionOrError(HttpCommandProcessor.java:111)
	at com.thoughtworks.selenium.HttpCommandProcessor.doCommand(HttpCommandProcessor.java:105)
	at com.thoughtworks.selenium.DefaultSelenium.open(DefaultSelenium.java:372)
	at testcases.TC_001.testUntitled1(TC_001.java:31)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- testUntitled1 -->
</testsuite> <!-- Test -->
